hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/table/TestHoodieMergeOnReadTable.java

```diff
@@ -605,6 +605,10 @@ public void testMetadataStatsOnCommit(Boolean rollbackUsingMarkers) throws Excep
       assertEquals(0, inserts);
       assertEquals(200, upserts);
 
+      if (!rollbackUsingMarkers) {
+        // we can do listing based rollback only when commit is completed
+        assertTrue(client.commit(instant2, statuses), "Commit should succeed");
```

Comment:
don't know what this test for?

Reply:
Please check https://github.com/apache/hudi/pull/13007#discussion_r2017704147

Reply:
This is just making sure that we have a commit when rollbackUsingMarkers is set to false. Listing based rollback only happens for completed commits